Runtime/OpenAIApi.cs

```diff
@@ -122,10 +122,9 @@ public OpenAIApi(string apiKey = null, string organization = null)
                     foreach (string line in lines)
                     {
                         var value = line.Replace("data: ", "");
-                        
-                        if (value.Contains("[DONE]")) 
+                        if (value.Contains("stop")) 
```

Comment:
Since `value` contains the entire chunk response, wouldn't this incorrectly set `isDone` to true if the delta content contained the word stop? I think a better solution would be to only set `isDone` when the value contains `[DONE]`. Or explicitly see if it contains `"finish_reason":"stop"`